src/components/FlowDesigner.container.jsx

```diff
@@ -19,6 +19,8 @@ import { setNodeTypes } from '../actions/nodeType.actions';
 export const FlowDesigner = React.createClass({
 	propTypes: {
 		children: PropTypes.node,
+		defs: PropTypes.arrayOf(PropTypes.element),
```

Comment:
in svg you can define many elements that you target to reuse in you document, like filter, patterns, gradients whatever.
I made this change so you can define your own reusable elements when initialising flow renderer, and reuse those element on your own component implementations.